### DISCUSSION TITLE
About the updates

### DISCUSSION THREAD

**Discussion by lkzMini** [Alex313031/thorium#626] [Q&A]
I noticed the browser is falling behind in updates. We are now at 122.0.6261.132... Can i ask why? When 123? Im just curious here.

**Comment by RocGod** [comment]
I think there is no auto-update as regular Chrome. @Alex313031 needs to fix some bugs, implementing some new features/enhancement and manually migrate to latest version. Usually Thorium is 1 version behind the current version of Chrome. Hope this make sense to you. Thank you.
